Expecto.netcore/Expecto.netcore.fsproj

```diff
@@ -0,0 +1,57 @@
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" />
+  <PropertyGroup>
+    <AssemblyName>Expecto.netcore</AssemblyName>
+    <Version>1.0.0-preview3</Version>
+    <OutputType>Library</OutputType>
+    <TargetFramework>netstandard1.6</TargetFramework>
+    <DebugType>pdbonly</DebugType>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)' == 'Release' ">
+    <DefineConstants>$(DefineConstants);RELEASE</DefineConstants>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="..\paket-files\logary\logary\src\Logary.Facade\Facade.fs" />
+    <Compile Include="..\Expecto\Expecto.fs" />
+    <Compile Include="..\Expecto\Expect.fs" />
+    <EmbeddedResource Include="**\*.resx" />
```

Comment:
don't forget to do it yourself
- https://github.com/dotnet/netcorecli-fsc/blob/master/examples/preview3/console/ExampleConsoleApp.fsproj#L12
- https://github.com/dotnet/netcorecli-fsc/blob/master/examples/preview3/console-crossgen/ExampleConsoleApp.fsproj#L13
- https://github.com/dotnet/netcorecli-fsc/blob/master/examples/preview3/lib/ExampleLibrary.fsproj#L11